Resources/public/mapbender.element.digitizer.js

```diff
@@ -998,7 +998,9 @@
                 schema.elementsTranslated = true;
             }
 
-            DataUtil.eachItem(widget.currentSettings.formItems, function(item) {
+            var formItems = [ $.extend(true,{},widget.currentSettings.formItems)[0]]; // Deep clone
```

Comment:
Why not just $.extend(true,[], widget.currentSettings.formItems) ?

Reply:
Well, that's what I did in cfc7282, but the root of `formItems` is supposed to be in array shape  in order to be processed by DataUtil.eachItem, while `$.extend` delivers a mere object.

Reply:
Mind the second parameter in my comment. In cfc7282 you use the object literal and not an array one.

Reply:
Ah, well, why not then. It's definitely more readable...

Reply:
Now...